fetch.bs

```diff
@@ -3342,86 +3345,149 @@ steps:
 
  <li><p>Let <var>response</var> be null.
 
+ <li><p>Let the <var>revalidating</var> and <var>partial</var> flags be unset.
+
  <li>
-  <p>If <var>httpRequest</var>'s
-  <a for=request>cache mode</a> is neither
-  "<code>no-store</code>" nor "<code>reload</code>", and there is a <em>complete</em>
-  <a for=/>response</a> in the HTTP cache for
-  <var>httpRequest</var> run these substeps:
-  <!-- XXX xref "HTTP cache" -->
+  <p>If <var>httpRequest</var>'s <a for=request>cache mode</a> is neither "<code>no-store</code>"
+  nor "<code>reload</code>", run these substeps:
 
   <ol>
    <li>
-    <p>If <var>httpRequest</var>'s <a for=request>cache mode</a> is
-    "<code>force-cache</code>" or "<code>only-if-cached</code>", then set
-    <var>response</var> to the <a for=/>response</a> in the HTTP cache for
-    <var>httpRequest</var>.
+    <p>Let <var>storedResponse</var> be the result of selecting a response from the HTTP cache,
```

Comment:
I meant that if you declare storedResponse upfront outside these substeps as "Let storedResponse be null" (to make it clearer what its scope is) you can rephrase this too.